test_ngtf.py

```diff
@@ -111,7 +112,8 @@ def run_tensorflow_pytests(venv_dir, build_dir, ngraph_tf_src_dir, tf_src_dir):
 
     command_executor([
         "python", test_script, "--tensorflow_path", tf_src_dir,
-        "--run_tests_from_file", test_manifest_file
+        "--run_tests_from_file", test_manifest_file, 
```

Comment:
We want this function to have a parameter to indicate whether to use a xml report or not. Then in the call site deside whether to use xml_report or not based on user input.

Reply:
Hi @avijit-nervana I already added an argument --xml_report which is an optional argument. If the user does not pass this argument, it will print the test result summary and if this argument is passed it will generate the xml report.